modules/investigating-worker-node-installation-issues.adoc

```diff
@@ -0,0 +1,200 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-installations.adoc
+
+[id="investigating-worker-node-installation-issues_{context}"]
+= Investigating worker node installation issues
+
+If you experience worker node installation issues, you can review the worker node status. Collect `kubelet.service`, `crio.service` journald unit logs and the worker node container logs for visibility into the worker node agent, CRI-O container runtime and Pod activity. Additionally, you can check the Ignition file and Machine API Operator functionality. If worker node post-installation configuration fails, check Machine Config Operator (MCO) and DNS functionality. You can also verify system clock synchronization between the bootstrap, master and worker nodes and validate certificates.
```

Comment:
missing a serial comma in the last sentence of this paragraph

Reply:
Resolved.